### DISCUSSION TITLE
NextCloudPi - bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nextcloudpi-v5.sh)"

### DISCUSSION THREAD

**Discussion by sebastjanp** [tteck/Proxmox#1038] [Help Needed] [answered]
Hello @tteck  :)

First, let me thank you for your great work at Proxmox helper scripts.

I'm using your NextCloudPi script that works great for setting all up, but it fails when started as an update script.

When I run 
`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nextcloudpi-v5.sh)"`

I got an error stating "No NextCloudPi installation Found!"

Please help

**Comment by tteck** [comment, marked as answer]
Where did you discover that method for updating NextcloudPi? https://github.com/tteck/Proxmox/issues/914#issuecomment-1376138884
Updates can be performed from the NextcloudPi admin panel.

**Comment by rollingafull** [comment]
> Hello @tteck :)
> 
> First, let me thank you for your great work at Proxmox helper scripts.
> 
> I'm using your NextCloudPi script that works great for setting all up, but it fails when started as an update script.
> 
> When I run `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nextcloudpi-v5.sh)"`
> 
> I got an error stating "No NextCloudPi installation Found!"
> 
> Please help

You can update your nextcloupi installatión from admin page at https://your_nextcloudpi_ip:4443.

Fron shell you can do 'ncp-update' for update nextcloudpi or use 'ncp-config' for update nextcloudpi, nextcloud, nextcloud-apps, and do other configurations like change forgoten passwords.
